ssg/build_yaml.py

```diff
@@ -1146,6 +1146,11 @@ def to_xml_element(self, env_yaml=None):
             # any deviations or stricter settings are applied by the rules in the profile.
             "software", "integrity", "integrity-software", "rpm_verification",
 
+            # Installing fapolicyd package creates a new user called fapolicyd
+            # and password_expiration (subgroup of accounts) rules configure parameters for
+            # all users in the system. So it should come first then accounts group.
+            "fapolicyd",
```

Comment:
The `fapolicyd` package is still installed prior to configuring password remediation.
This happens because they are in different branches, :( 

```
./linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing
./linux_os/guide/services/fapolicyd/package_fapolicyd_installed
```

System and Service groups would have to be swapped.

Reply:
And indeed swapping the groups fixed this particular issue.

Reply:
"historically" the `system` group was placed before the  `services` group, and we kept it that way:
https://github.com/ComplianceAsCode/content/pull/4900

I'm not sure what other consequences swapping `system` and `services` would do...